src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/ClaimIssuedStaffNotificationService.java

```diff
@@ -30,47 +28,46 @@
     private final EmailService emailService;
     private final StaffEmailProperties staffEmailProperties;
     private final ClaimIssuedStaffNotificationEmailContentProvider provider;
-    private final PDFServiceClient pdfServiceClient;
-    private final SealedClaimContentProvider sealedClaimContentProvider;
-    private final DefendantPinLetterContentProvider defendantPinLetterContentProvider;
-    private final LegalSealedClaimContentProvider legalSealedClaimContentProvider;
+    private final DefendantPinLetterPdfService defendantPinLetterPdfService;
+    private final SealedClaimDocumentService sealedClaimDocumentService;
 
     @Autowired
     public ClaimIssuedStaffNotificationService(
         final EmailService emailService,
         final StaffEmailProperties staffEmailProperties,
         final ClaimIssuedStaffNotificationEmailContentProvider provider,
-        final PDFServiceClient pdfServiceClient,
-        final SealedClaimContentProvider sealedClaimContentProvider,
-        final DefendantPinLetterContentProvider defendantPinLetterContentProvider,
-        final LegalSealedClaimContentProvider legalSealedClaimContentProvider
+        final DefendantPinLetterPdfService defendantPinLetterPdfService,
+        final SealedClaimDocumentService sealedClaimDocumentService
     ) {
         this.emailService = emailService;
         this.staffEmailProperties = staffEmailProperties;
         this.provider = provider;
-        this.pdfServiceClient = pdfServiceClient;
-        this.sealedClaimContentProvider = sealedClaimContentProvider;
-        this.defendantPinLetterContentProvider = defendantPinLetterContentProvider;
-        this.legalSealedClaimContentProvider = legalSealedClaimContentProvider;
+        this.defendantPinLetterPdfService = defendantPinLetterPdfService;
+        this.sealedClaimDocumentService = sealedClaimDocumentService;
     }
 
-    public void notifyStaffClaimIssued(Claim claim, Optional<String> defendantPin, String submitterEmail) {
+    public void notifyStaffClaimIssued(final Claim claim,
```

Comment:
could you move the claim parameter to the next line and the closing brace to the next line please?